/*************************************************************************
* ABSOLUTELY UNDER NO CONDITION SHOULD YOU INCLUDE THIS FILE DIRECTLY    *
* INSTEAD; YOU SHOULD INCLUDE RepeatManagerKeyAction.pb.h                *
*                                                                        *
* This file is auto generated, DO NOT EDIT!                              *
* To Generate this file, execute create_keyaction_dependencies file      *
*************************************************************************/

enum KEY_EVENT_ACTION
{
    INVALID_KEY_EVENT_ACTION = -1;
    KEY_EVENT_PLAY = 0;
    KEY_EVENT_PAUSE = 1;
    KEY_EVENT_STOP = 2;
    KEY_EVENT_PREVIOUS_TRACK = 3;
    KEY_EVENT_NEXT_TRACK = 4;
    KEY_EVENT_THUMBS_UP = 5;
    KEY_EVENT_THUMBS_DOWN = 6;
    KEY_EVENT_POWER = 7;
    KEY_EVENT_BOOKMARK = 8;
    KEY_EVENT_MUTE = 9;
    KEY_EVENT_VOLUME_UP = 10;
    KEY_EVENT_VOLUME_DOWN = 11;
    KEY_EVENT_PRESET_1 = 12;
    KEY_EVENT_PRESET_2 = 13;
    KEY_EVENT_PRESET_3 = 14;
    KEY_EVENT_PRESET_4 = 15;
    KEY_EVENT_PRESET_5 = 16;
    KEY_EVENT_PRESET_6 = 17;
    KEY_EVENT_AUX_INPUT = 18;
    KEY_EVENT_SHUFFEL_ON = 19;
    KEY_EVENT_SHUFFEL_OFF = 20;
    KEY_EVENT_REPEAT_OFF = 21;
    KEY_EVENT_REPEAT_ONE = 22;
    KEY_EVENT_REPEAT_ALL = 23;
    KEY_EVENT_PLAY_PAUSE = 24;
    KEY_EVENT_ADD_FAVORITE = 25;
    KEY_EVENT_REMOVE_FAVORITE = 26;
    KEY_EVENT_BLUETOOTH = 27;
    KEY_EVENT_VOLUME_DOWN_PRESET_1 = 28;
    KEY_EVENT_VOLUME_DOWN_PRESET_2 = 29;
    KEY_EVENT_PRESET_1_VOLUME_DOWN = 30;
    KEY_EVENT_PRESET_2_VOLUME_DOWN = 31;
    KEY_EVENT_PRESET_3_VOLUME_DOWN = 32;
    KEY_EVENT_PRESET_4_VOLUME_DOWN = 33;
    KEY_EVENT_PRESET_5_VOLUME_DOWN = 34;
    KEY_EVENT_PRESET_6_VOLUME_DOWN = 35;
    KEY_EVENT_PRESET_1_VOLUME_UP = 36;
    KEY_EVENT_PRESET_2_VOLUME_UP = 37;
    KEY_EVENT_PRESET_3_VOLUME_UP = 38;
    KEY_EVENT_PRESET_4_VOLUME_UP = 39;
    KEY_EVENT_PRESET_5_VOLUME_UP = 40;
    KEY_EVENT_PRESET_6_VOLUME_UP = 41;
    KEY_EVENT_PRESET_1_2_3_4 = 42;
    KEY_EVENT_LONG_PRESET_5 = 43;
    KEY_EVENT_EXTRA_LONG_PRESET5 = 44;
    KEY_EVENT_PLAY_PRESET_1_2 = 45;
    KEY_EVENT_LONG_PLAY_PRESET_1_2 = 46;
    KEY_EVENT_LONGEST_PRESET_5 = 47;
}
